specs/guard-jwt.spec/src/main/scripts/io/aklivity/zilla/specs/guard/jwt/config/guard-issue-68.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "test",
+    "guards":
+    {
+        "jwt0":
+        {
+            "type": "jwt",
+            "options":
+            {
+                "issuer": "https://auth.example.com",
+                "audience": "https://api.example.com",
+                "keys": "https://auth.example.com/.well_known/jwks.json",
+                "challenge": 30
+            }
+        }
+    }
+}
```

Comment:
Please name this file as if it had been present before the issue was filed, rather than tying it back to the issue number.

For example, `guard-keys-dynamic.json` rather than `guard-issue-68.json`.